### PR TITLE
Fix up theme to work with template updates

### PR DESCRIPTION
**Pull request by chrisbowley**
Needs some clean up!

### CODE REVIEW

File: fr_theme/index.html [hunk 1]
```diff
@@ -29,22 +29,25 @@
       </ul>
 
       <div class="container">
-        <header>
-          {{include src="../../header.hbs"}}
-        </header>
```
Comment (rblalock) [inline fr_theme/index.html:34]: There a reason to add this back in?

**Review by rblalock — CHANGES_REQUESTED** [review]
Screen Shot 2021-06-24 at 4 19 18 PM
Screen Shot 2021-06-24 at 4 20 40 PM
Screen Shot 2021-06-24 at 4 21 12 PM
Screen Shot 2021-06-24 at 4 21 20 PM

Not sure why but it doesn't seem like the CSS is applied.

**Comment by chrisbowley** [comment]
I cloned a fresh copy of this repo and re-tested this branch and everything seems fine. Here's what I did:

* Clone repo and cd into it
* `git checkout update`
* `npm install`
* Build the changelog using the custom there: `npm run build -- fishrules --theme-dir=./fr_theme`
* Run a local HTTP server to test: `cd dist && python -m SimpleHTTPServer 8080`
* Open browser at http://localhost:8080

Screenshot 2021-06-25 at 10 10 30

**Comment by rblalock** [comment]
`changelog-gen updates.fishrulesapp.com --theme-dir ./fr_theme --watch`

I was using this .. but i guess that doesn't work anymore?

**Review by rblalock — APPROVED** [review]

**Pull request merged**